src/components/tasks/card/index.tsx

```diff
@@ -230,7 +230,7 @@ const Card: FC<CardProps> = ({
     const hasIssueAssignee = () => cardDetails.github?.issue.assignee ?? false;
     const hasTaskAssignee = () => cardDetails.assignee ?? false;
     const isIssueClosed = () => cardDetails.github?.issue?.status === 'closed';
-    const isTaskComplete = () => cardDetails.status === COMPLETED;
+    const isTaskComplete = () => cardDetails.status === DONE;
```

Comment:
This line seems to be uncovered as you provide this coverage
![Uploading image.png…]()